pkg/github/runtime_release.go

```diff
@@ -17,6 +18,18 @@ const (
 	runtimeRepo  = "spiceai"
 )
 
+func CheckForLatestVersion() {
```

Comment:
As this is CLI specific it should probably go in something like `pkg/cli/version.go`

Reply:
It should also return the error, which should be checked by the caller. I.e.

```go
func checkForLatestVersion() error {
```

Reply:
Got your point. Will do the changes now